### PR TITLE
[Kotlin Migration] LrnCardQueue 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] LrnCardQueue

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]
LGTM. Kicked off CI

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt [hunk 1]
```diff
@@ -14,45 +14,35 @@
...
  ****************************************************************************************/
 
-package com.ichi2.libanki.sched;
+package com.ichi2.libanki.sched
 
-import java.util.Collections;
+import java.util.*
 
-class LrnCardQueue extends CardQueue<LrnCard> {
+internal class LrnCardQueue(sched: AbstractSched?) : CardQueue<LrnCard?>(sched) {
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:21]: Why internal?

Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:21]: <LrnCard>. It's not supposed to ever be a null card

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt [hunk 2]
```diff
@@ -14,45 +14,35 @@
...
-    }
-
-
-    @Override
-    public void clear() {
-        super.clear();
-        mIsFilled = false;
-    }
+    val firstDue: Long
+        get() = queue.first!!.due
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:38]: I believe previous modification will allow to remove the `!!`

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:21]: Error: 'public' subclass exposes its 'public/*package*/' supertype CardQueue,

Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:21]: Thanks, will make the required changes.

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt (hunk 2 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/LrnCardQueue.kt:38]: Yup, done locally.

**Comment by codingtosh** [comment]
@Arthur-Milchior 
Seems there has been a premature merge, I had not pushed the changes regarding nullability yet as I was wanted to know whether or not to keep the class internal? Please confirm what should be done, I'll make the required changes and upload a followup PR for this shortly.

**Comment by Arthur-Milchior** [comment]
Oops, my bad.
Please send a PR with non-null and it'll be all right

**Pull request merged**